### DISCUSSION TITLE
post comment return 500 internal server error

### DISCUSSION THREAD

**Discussion by chadimessmer** [VirtusLab-Open-Source/strapi-plugin-comments#112] [Q&A]
Hey everybody,

I'm trying to use this plugin but when I try to post a comment the server answers "500 Internal Server Error"

I'm sending the post request to https://www.mystrapiapp.com/api/comments/api::chatroom.chatroom:1 

with this message:  
{
  "author": {
      "id": "123456",
      "name": "Joe Doe",
      "email": "jdoe@sample.com",
  },
	"content": "My sample response",
}

this returns error  500. 
Every other path return 403 Method not allowed

I tried with postman / axios / graphql  and always the same answer...

I have strapi V4 with strapi-plugin-comments 2.0.3

Any idea what I'm doing wrong ?

thank you

**Comment by cyp3rius** [comment]
Created issue #120. Closing this discussion.
